css/obBase.css

```diff
@@ -132,8 +133,24 @@ form {
 
 #sideBar {
   position: absolute;
-  right: 0;
+  right: -180px;
+  top: 50px;
+  width:220px;
+  background-color: #252525;
+  height: calc(100% - 50px);
   z-index: 2;
+  -webkit-transition: all 0.5s ease;
+  -moz-transition: all 0.5s ease;
+  -o-transition: all 0.5s ease;
+  transition: all 0.5s ease;
```

Comment:
Don't use all, since in this case we only want specific properties to transition, make a rule for just those properties.

I think all was used elsewhere for elements that had modifier classes that added various new properties.